examples/prometheus/prometheus.yaml

```diff
@@ -48,10 +53,12 @@ objects:
     name: prometheus
     namespace: "${NAMESPACE}"
   spec:
+    host: "${PROMETHEUS_HOSTNAME}"
     to:
       name: prometheus
     tls:
       termination: Reencrypt
+      destinationCACertificate: "${SERVICE_SIGNER_CERT}"
```

Comment:
@zgalor can you remove this? You can keep it locally if for some reason you still need it.
Let's move forward with just the additional arguments and the `PROMETHEUS_HOSTNAME`.